protocol.html

```diff
@@ -807,6 +816,7 @@ <h3 property="schema:name" content="Notification Channel Data Model">Notificatio
                   <ul>
                     <li id="notify-channel-id">One <code>id</code> property to identify the notification channel.</li>
                     <li id="notify-channel-type">One <code>type</code> property to state the <a href="#notification-channel-types" rel="rdfs:seeAlso">notification channel type</a>.</li>
+                    <li id="notify-channel-authScheme">One <code>authScheme</code> property to state the authentication scheme used for the channel.</li>
```

Comment:
The #authentication-scheme section should describe that the absence of this property is required (or not). If present, it would entail that the schemes are known, including a scheme value such as "None" meaning that authentication is not required.

I presume "None" will map to `notify:None` .. but we mean something like `notify:AuthenticationSchemeIsNotUsed`?